### PR TITLE
docs: mention third-party-capital and aurora for tpc composables

### PR DESCRIPTION
**Pull request by huang-julien**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Mention third-party-capital

### CODE REVIEW

**Comment by harlan-zw** [comment]
Maybe it would be better to do an actual link if users want to check out the project?

File: docs/content/scripts/analytics/google-analytics.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ links:
     size: xs
 ---
 
+::tip
+This composable is generated with `GoogleChromeLabs/third-party-capital` in collaboration with the Aurora team.
```
Comment (kara) [inline docs/content/scripts/analytics/google-analytics.md]:
```suggestion
This composable is generated with `GoogleChromeLabs/third-party-capital` in collaboration with the [Chrome Aurora team](https://developer.chrome.com/docs/aurora).
```

File: docs/content/scripts/tracking/google-tag-manager.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ links:
   size: xs
 ---
 
+::tip
+This composable is generated with `GoogleChromeLabs/third-party-capital` in collaboration with the Aurora team.
```
Comment (kara) [inline docs/content/scripts/tracking/google-tag-manager.md]:
```suggestion
This composable is generated with `GoogleChromeLabs/third-party-capital` in collaboration with the [Chrome Aurora team](https://developer.chrome.com/docs/aurora).
```

**Review by kara — APPROVED** [review]
LGTM, small nit to add a link. Thanks so much for taking the time to do this :)

**Pull request merged**